concrete/controllers/single_page/dashboard/reports/forms/legacy.php

```diff
@@ -85,7 +85,14 @@ public function csv()
             foreach ($questions as $questionId => $question) {
                 if ($question['inputType'] == 'checkboxlist') {
                     $options = explode('%%', $question['options']);
-                    $subanswers = explode(',', $answerSet['answers'][$questionId]['answer']);
+
+                    $subanswers = [];
+                    if (isset($answerSet['answers']) &&
```

Comment:
Please remark that there's no need to check with `isset()` for nested variables, it's enough to have just
`isset($answerSet['answers'][$questionId]['answer']))`

See for example https://3v4l.org/kEsjR